specification/redis/resource-manager/Microsoft.Cache/stable/2020-12-01/examples/RedisCacheGet.json

```diff
@@ -36,17 +37,20 @@
             {
               "sslPort": 15000,
               "nonSslPort": 13000,
-              "isMaster": true
+              "isMaster": true,
```

Comment:
isMaster and isPrimary are not present in other examples - RedisCacheList, RedisCacheUpdate. Can you please update them